src/plugins/theme/spacing.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Spacing - used in various classes such as margin and padding; individual values taken from Responsive Spacing map
```

Comment:
I replaced all the spacing values to use our individual steps from the responsive spacing map as a way to replace our scss `get-responsive-spacing-value` function

Reply:
Also added from 1-10px for finetuning margin as things - the small steps gets used often.